README.md

```diff
@@ -0,0 +1,166 @@
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started ](#-getting-started-)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors ](#-authors-)
+- [🔭 Future Features ](#-future-features-)
+- [🤝 Contributing ](#-contributing-)
+- [⭐️ Show your support ](#️-show-your-support-)
+- [🙏 Acknowledgments ](#-acknowledgments-)
+- [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 ror-budgetapp  <a name="about-project"></a>
+
+**ror-budgetapp** is a mobile web application built using Ruby on Rails that helps users manage their spending. With this app, users can register an account and log in, ensuring that their data remains private and secure.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <ul>
+    <li><a href="https://guides.rubyonrails.org/getting_started.html">Ruby on Rails</a></li>
+  </ul>
+
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Splash screen.**
+- **A login page.**
+- **A registration page.**
+- **A categories page with all the transactions that belongs to that category.**
+- **A detal page with all the categories.**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Live demo link](https://expense-tracker-frvr.onrender.com)
```

Comment:
- [ Required ] Good job adding a presentation video, however, the length of the video is a bit longer than required. The project requirements state that the length of the presentation video should not be longer than `5` minutes, yet your presentation is `6` minutes longer. It would be nice if you record a 5-minute video presentation since this is the length that most recruiters request when vetting developers for jobs. Please see the screenshot below and rectify this.

## Project requirements
![Screenshot from 2023-08-09 02-45-00](https://github.com/Naledi-Dikgale/ror-budgetapp/assets/16392046/bcb7b62c-804e-4f55-8337-e14e5fb466fa)